src/store/Business/getters.ts

```diff
@@ -0,0 +1,161 @@
+import { CorpTypeCd, EntityState } from '@/enums'
+import { BusinessIF, BusinessWarningIF } from '@/interfaces'
+import { DateUtilities } from '@/services/'
+
+export default {
+  /** The business number (aka Tax ID). */
+  getBusinessNumber (state: BusinessIF): string {
+    return state.taxId
+  },
+
+  /** The business warnings list. */
+  getBusinessWarnings (state: BusinessIF): Array<BusinessWarningIF> {
+    return state.warnings
+  },
+
+  /** The business state. */
+  getBusinessState (state: BusinessIF): EntityState {
+    return state.state
+  },
+
+  /** The founding date. */
+  getFoundingDate (state: BusinessIF): Date {
+    return DateUtilities.apiToDate(state.foundingDate)
+  },
+
+  /** The business identifier (aka Incorporation Number). */
+  getIdentifier (state: BusinessIF): string {
+    return state.identifier
+  },
+
+  /** The last address change date. */
+  getLastAddressChangeDate (state: BusinessIF): string {
+    return state.lastAddressChangeDate
+  },
+
+  /** The last annual report change date. */
+  getLastAnnualReportDate (state: BusinessIF): string {
+    return state.lastAnnualReportDate
+  },
+
+  /** The last director change date. */
+  getLastDirectorChangeDate (state: BusinessIF): string {
+    return state.lastDirectorChangeDate
+  },
+
+  /** The legal name. */
+  getLegalName (state: BusinessIF): string {
+    return state.legalName
+  },
+
+  /** The legal type. */
+  getLegalType (state: BusinessIF): CorpTypeCd {
+    return state.legalType
+  },
+
+  /** The state filing URL (may be null). */
+  getStateFilingUrl (state: BusinessIF): string {
+    return state.stateFiling
+  },
+
+  /** Is true of the business has a court order filing */
+  hasCourtOrders (state: BusinessIF): boolean {
+    return state.hasCourtOrders
+  },
+
+  /** Is True if a firm has at least one "compliance" warning. */
+  hasComplianceWarning (_state: BusinessIF, getters): boolean {
```

Comment:
Using `_state` so this unused parameter doesn't cause a warning.

Note the use of additional parameter `getters` (from this module). There's also `rootState` and `rootGetters` (from rest of store).